src/nvme/api-types.h

```diff
@@ -199,7 +199,7 @@ struct nvme_format_nvm_args {
  */
 struct nvme_ns_mgmt_args {
 	__u32 *result;
-	struct nvme_id_ns *ns;
+	struct nvme_ns_mgmt_host_sw_specified *data;
```

Comment:
As far I can tell, this breaks the API, although the types have the same types (pointer). The caller might hand in a `ns` pointer and the library expects a `nvme_ns_mgmt_host_sw_specified` type.